### PR TITLE
Initial installs

### PR DESCRIPTION
**Pull request by RyanJ2463**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -64,3 +64,13 @@ group :development do
   # gem "spring"
 end
 
+
+gem "rspec-rails", "~> 5.1"
```
Comment (VonTeacher) [inline Gemfile:68]: Consider nesting this in `development / test` as per the docs.
```
group :development, :test do
  gem 'rspec-rails', '~> x.x.x'
end
```

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -13,3 +13,4 @@
  *= require_tree .
  *= require_self
  */
+ @import 'bootstrap';
```
Comment (VonTeacher) [inline app/assets/stylesheets/application.scss:16]: You're going to see this a lot. Please make sure you have settings in your IDE to add a new line at the end of every file.

File: app/models/song.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class Song < ApplicationRecord
+    belongs_to :user
```
Comment (VonTeacher) [inline app/models/song.rb:2]: This indent is 4 spaces. Ruby convention is to use 2. Check your IDE to ensure your settings are accurate.

File: app/views/home/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<%= react_component 'App', {
+  logged_in: user_signed_in?,
+  current_user: current_user,
+  new_user_route: new_user_registration_path,
+  sign_in_route: new_user_session_path,
+  sign_out_route: destroy_user_session_path
+} %>
```
Comment (VonTeacher) [inline app/views/home/index.html.erb:7]: EOF new line, please.

File: db/seeds.rb [hunk 5]
```diff
@@ -5,3 +5,49 @@
...
+songs.each do |attribute|
+    user = User.find 1
+    user.songs.create attribute
+    puts "creating #{attribute}"
+end
+
+users.each do |attribute|
+    User.create attribute
+    puts "creating #{attribute}"
+end
```
Comment (VonTeacher) [inline db/seeds.rb:53]: Let's fix indent and EOF newline.

**Review by VonTeacher — CHANGES_REQUESTED** [review]
Suggested a few changes that will help our consistency going forward.

**Review by WalkerTexas-coder — APPROVED** [review]
LGTM!

File: app/assets/stylesheets/application.scss [hunk 6]
```diff
@@ -13,3 +13,5 @@
...
+
```
Comment (VonTeacher) [inline app/assets/stylesheets/application.scss:17]: Not sure what happened, but not a blocker. :D

File: db/migrate/20220420204027_create_songs.rb [hunk 7]
```diff
@@ -0,0 +1,11 @@
+class CreateSongs < ActiveRecord::Migration[7.0]
+  def change
+    create_table :songs do |t|
+      t.string :title
+      t.string :artist
```
Comment (VonTeacher) [inline db/migrate/20220420204027_create_songs.rb:5]: We don't have to address this now, but remind me to talk about database level settings that can help us protect against `nil`/`empty` fields.

File: db/seeds.rb [hunk 8]
```diff
@@ -5,3 +5,48 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+
+
+users = [
```
Comment (VonTeacher) [inline db/seeds.rb:11]: Optional, since we won't touch this much, but keep eyes out for indentation going forward.

**Review by VonTeacher — APPROVED** [review]
Looks good. 👍  Going forward, smaller commits will be easier for review and speed up our dev process.

**Pull request merged**